ecommerce-cart/src/test/kotlin/com/kotato/context/ecommerce/infrastructure/CqrsCommandBusTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.kotato.context.ecommerce.infrastructure
+
+import com.kotato.EcommerceApplication
+import com.kotato.cqrs.domain.command.Command
+import com.kotato.cqrs.domain.command.CommandBus
+import com.kotato.cqrs.domain.command.CommandHandler
+import org.axonframework.commandhandling.NoHandlerForCommandException
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import javax.inject.Inject
+import javax.inject.Named
+import kotlin.test.assertFailsWith
+
+@SpringBootTest(classes = arrayOf(EcommerceApplication::class))
```

Comment:
inherit this test class from ContextStarterTest  instead of using custom context